### ISSUE TITLE
Cutscene with Elysia is Silent plus other issues I've encountered from before.

### ISSUE THREAD

**Issue by FRCommander** [BuIlDaLiBlE/BetterHI3Launcher#30]
Hello. I just wanted to say that I was able to try this not long ago and this is actually really great. However I'm afraid I'm having some technical issues here. Before another update was given (Before this event with Seele) my problem was that whenever I'm in a cutscene usually main story, it was still the same thing just a blurry background but I can still hear the voices. But after the update I decided to try out the event and I was able to see the characters in the cutscenes now. Though I only tried it in the event so far and haven't touched the Main Story Cutscene yet. But I guess I can let it slide. My next issue is that Whenever I talk with Elysia in the Elysian Realm, I can't seem to hear her and I also can't here Mei talking too. Their conversation is silent but when I talk or initiate a cutscenes with the others they all have voices. I tried to repair game files, and downloaded full cache and still nothing works. Maybe there's a bug I guess? Anyways this is all I'm concerned for now. Overall I'm really liking this BetterHI3Launcher. I might recommend this to my friend who plays in PC as well.

**Comment by BuIlDaLiBlE** [comment]
Thanks for the feedback! Glad you're enjoying the launcher.
Regarding Elysia, it's just that there is almost no voice for her right now. Mei also has some lines unvoiced, that's all intentional behavior. 5.1 should have more lines voiced hopefully.

**Comment by FRCommander** [comment]
Hello there. I apologise if I'm submitting my feedback here but
unfortunately I forgot my password and when I request to reset it I keep
getting the "Unable to Verify Captcha" Message. I have decided to put it
here as this is my last resort for submitting feedback. So it's about the
latest update for Honkai Impact and as I was updating I always notice that
the update sometimes just stop and sometimes I let it do its thing for 6
hours and the update hasn't moved since. I tried restarting both my laptop
and wifi and it does resume and it just repeats the same process.
Apparently after a few retries the update was finally moving without
interruption. And I was able to update my game. But when it was
verifying in game files, an error occurred. So I went back to the launcher
and decided to fix the files. And unfortunately 770 files were
corrupted/missing and I decided to let it fix. But for some reason it was
acting the same way as it was during the update like fixing it then it just
stops there for a really long time. So once again I tried to restart it but
this time I got this error now (I'll include a picture. Refer to it there).
As I have no experience in this kind of errors I am asking if anyone else
who is using this launcher is having the same experience or do I need to
reinstall it again? I also let the original HI3 game download again since
its telling me to get a new launcher. Thank you for your time.[image:
image_2021-12-01_172027.png]
